planning/behavior_path_planner/autoware_behavior_path_static_obstacle_avoidance_module/include/autoware/behavior_path_static_obstacle_avoidance_module/helper.hpp

```diff
@@ -321,6 +321,19 @@ class AvoidanceHelper
     });
   }
 
+  bool isFeasible(const AvoidLineArray & shift_lines) const
+  {
+    const auto JERK_BUFFER = 0.1;  // [m/sss]
```

Comment:
nit: constexpr double might be better
```suggestion
    constexpr double JERK_BUFFER = 0.1;  // [m/sss]
```